cpp/arcticdb/version/versioned_engine.hpp

```diff
@@ -121,7 +121,8 @@ class VersionedEngine {
         bool validate_index
     ) = 0;
 
-    virtual std::pair<VersionedItem, std::vector<AtomKey>> write_individual_segment(
+    /** Test-specific cut-down version of write_versioned_dataframe_internal */
```

Comment:
This is only used in tests?

Reply:
Yes, and it's not in `arcticc`, which makes me believe it was added because `write_versioned_dataframe_internal` picked up some new behaviour that we don't want in tests?